www/contribute/issues.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: contribute-help
+title: Apache Cordova Reporting Issues

```

Comment:
Where does this title appear? Should there be a hyphen or something between the project name and page topic?